ansible/roles/topology/README.md

```diff
@@ -0,0 +1,14 @@
+topology
+========
+
+Templates out /etc/slurm/topology.conf file based on an Openstack project for use by
+Slurm's [topology/tree plugin.](https://slurm.schedmd.com/topology.html) Models
+project as tree with a heirarchy of:
+
+Project -> Availability Zones -> Hypervisors -> VMs
+
+Role Variables
+--------------
+
+- `topology_topology_nodes: []`: Required list[str]. List of nodes to include in topology tree. Must be set to include all compute nodes in Slurm cluster. Default `[]`.
```

Comment:
```suggestion
- `topology_topology_nodes: []`: Required list of strs. List of nodes to include in topology tree. Must be set to include all compute nodes in Slurm cluster. Default `[]`.
```

We don't really use that formal typing anywhere else (maybe we should!)

Reply:
And, this is actually a list of "inventory hostnames" right? I'm fine with the role var name but if I'm right we shold definitlely clarify that.